rollup.config.js

```diff
@@ -49,11 +54,23 @@ module.exports = [
                 react: 'React'
             }
         },
+        // core-js polyfills are imported for node/esm compatibility, so core-js@3.11.x is
+        // technically a peer for consuming the UMD output those environments e.g. node, but bundled
+        // for running in-browser
+        external: [/^core\-js/],
         plugins: [
             PeerDepsExternal(),
-            Resolve(),
+            NodeResolve(),
             Commonjs(),
-            Babel({ exclude: ['node_modules/**'] }),
+            Babel({
+                exclude: ['node_modules/**'],
+                // babelHelpers and useBuiltIns together result in our used polyfills e.g.
+                // regenerator-runtime bundled in the output UMD, so it's directly usable
```

Comment:
This does bloat the umd bundle pretty significantly, for whatever that's worth, roughly 11 --> 18kb

Reply:
I am assuming that either the UMD build was broken before this PR, or that it already contained these helpers. Does that seem right?  If so, I think we're all good.

Reply:
Yea, that checks out. Just ran the demo on a build from the current main branch, got the same `ReferenceError: regeneratorRuntime is not defined` crash that all this stuff works around